.github/ISSUE_TEMPLATE.md

```diff
@@ -6,6 +6,10 @@
 
 **Environment**
 
+* Prometheus Operator version:
+
+	insert image tag or output of `./operator version` here
```

Comment:
ah totally true, I quickly typed `./operator version` and saw the version printed but failed to see the error causing the operator to exit so I assumed it was implemented. Sure, I can add a flag to do that. Otherwise, we can ask users to just write the image tag or commit SHA they used. WDYT?

Reply:
> Otherwise, we can ask users to just write the image tag or commit SHA they used. WDYT?

Also fine with me!

Reply:
Awesome. Let's do that and simplify our lives rather than make a one-off arg parser or something.

Reply:
done!